compiler/src/model/metamodel.ts

```diff
@@ -378,12 +378,30 @@ export class Deprecation {
   description: string
 }
 
+export class Availability {
```

Comment:
Rename to `Availabilities` (plural) since it's a collection of named individual `Availability`?